assets/xml/objects/object_kanban.xml

```diff
@@ -1,18 +1,19 @@
 ﻿<Root>
+    <!-- Assets for cuttable signs-->
     <File Name="object_kanban" Segment="6">
-        <DList Name="object_kanban_DL_000C30" Offset="0xC30" />
-        <DList Name="object_kanban_DL_000CB0" Offset="0xCB0" />
-        <DList Name="object_kanban_DL_000DB8" Offset="0xDB8" />
-        <DList Name="object_kanban_DL_000E78" Offset="0xE78" />
-        <DList Name="object_kanban_DL_000F38" Offset="0xF38" />
-        <DList Name="object_kanban_DL_000FF8" Offset="0xFF8" />
-        <DList Name="object_kanban_DL_0010B8" Offset="0x10B8" />
-        <DList Name="object_kanban_DL_0011C0" Offset="0x11C0" />
-        <DList Name="object_kanban_DL_0012C8" Offset="0x12C8" />
-        <DList Name="object_kanban_DL_0013D0" Offset="0x13D0" />
-        <DList Name="object_kanban_DL_001488" Offset="0x1488" />
-        <DList Name="object_kanban_DL_001540" Offset="0x1540" />
-        <DList Name="object_kanban_DL_001630" Offset="0x1630" />
-        <Texture Name="object_kanban_Tex_0016B0" OutName="tex_0016B0" Format="i8" Width="16" Height="16" Offset="0x16B0" />
+        <DList Name="gSignSetupDL" Offset="0xC30" />
```

Comment:
```suggestion
        <DList Name="gSignMaterialDL" Offset="0xC30" />
```
Generally the two dl sets are named `Material` (which sets everything up but does not have any vertex information) and `Model` (which does hold the vertex information). I'd probably say add `Model` to the other dls (excpet `SignParticle`) as well.
Something like `gSignUpperLeftModelDL`.

Reply:
oh yea i was wondering whether that one was a material or something but usually (as in the ones ive seen lol) its just one material and on model so i got a bit confused but yea on it